pep-8001.rst

```diff
@@ -13,7 +13,7 @@ Author: Brett Cannon <brett@python.org>,
         Tal Einat <taleinat@gmail.com>,
         Tim Peters <tim.peters@gmail.com>,
         Zachary Ware <zachary.ware@gmail.com>
-Status: Draft
+Status: Accepted
```

Comment:
I changed the status to Accepted.
Should it be Final? or Active?

Reply:
This is good. We'll mark it Final tomorrow when the dust settles. Active means "this is not meant to be finished".

Reply:
I think "Accepted" as "Final" is meant to be when something has been "committed", so until the vote starts I don't think "Final" is accurate.